solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1542 @@
+    );
+    event MovedFundsSweepTimeoutUpdated(uint32 movedFundsSweepTimeout);
+
+    event MovedFundsSweepTimeoutSlashingAmountUpdateStarted(
+        uint96 newMovedFundsSweepTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutSlashingAmountUpdated(
+        uint96 movedFundsSweepTimeoutSlashingAmount
+    );
+
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovedFundsSweepTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movedFundsSweepTimeoutNotifierRewardMultiplier
+    );
+
+    event WalletCreationPeriodUpdateStarted(
+        uint32 newWalletCreationPeriod,
+        uint256 timestamp
+    );
+    event WalletCreationPeriodUpdated(uint32 walletCreationPeriod);
+
+    event WalletCreationMinBtcBalanceUpdateStarted(
+        uint64 newWalletCreationMinBtcBalance,
+        uint256 timestamp
+    );
+    event WalletCreationMinBtcBalanceUpdated(
+        uint64 walletCreationMinBtcBalance
+    );
+
+    event WalletCreationMaxBtcBalanceUpdateStarted(
+        uint64 newWalletCreationMaxBtcBalance,
+        uint256 timestamp
+    );
+    event WalletCreationMaxBtcBalanceUpdated(
+        uint64 walletCreationMaxBtcBalance
+    );
+
+    event WalletClosureMinBtcBalanceUpdateStarted(
+        uint64 newWalletClosureMinBtcBalance,
+        uint256 timestamp
+    );
+    event WalletClosureMinBtcBalanceUpdated(uint64 walletClosureMinBtcBalance);
+
+    event WalletMaxAgeUpdateStarted(uint32 newWalletMaxAge, uint256 timestamp);
+    event WalletMaxAgeUpdated(uint32 walletMaxAge);
+
+    event WalletMaxBtcTransferUpdateStarted(
+        uint64 newWalletMaxBtcTransfer,
+        uint256 timestamp
+    );
+    event WalletMaxBtcTransferUpdated(uint64 walletMaxBtcTransfer);
+
+    event WalletClosingPeriodUpdateStarted(
+        uint32 newWalletClosingPeriod,
+        uint256 timestamp
+    );
+    event WalletClosingPeriodUpdated(uint32 walletClosingPeriod);
+
+    event FraudChallengeDepositAmountUpdateStarted(
+        uint256 newFraudChallengeDepositAmount,
+        uint256 timestamp
+    );
+    event FraudChallengeDepositAmountUpdated(
+        uint256 fraudChallengeDepositAmount
+    );
+
+    event FraudChallengeDefeatTimeoutUpdateStarted(
+        uint256 newFraudChallengeDefeatTimeout,
+        uint256 timestamp
+    );
+    event FraudChallengeDefeatTimeoutUpdated(
+        uint256 fraudChallengeDefeatTimeout
+    );
+
+    event FraudSlashingAmountUpdateStarted(
+        uint96 newFraudSlashingAmount,
+        uint256 timestamp
+    );
+    event FraudSlashingAmountUpdated(uint96 fraudSlashingAmount);
+
+    event FraudNotifierRewardMultiplierUpdateStarted(
+        uint256 newFraudNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event FraudNotifierRewardMultiplierUpdated(
+        uint256 fraudNotifierRewardMultiplier
+    );
+
```

Comment:
The type of `fraudNotifierRewardMultiplier` is `uint32`:
```
uint32 fraudNotifierRewardMultiplier;
```
Let's change the type in every occurrence of this parameter (functions, events).

Reply:
Done